jest.config.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+const ENABLE_COVERAGE = !!process.env.CI || !!process.env.COVERAGE;
+
+module.exports = {
+  collectCoverage: ENABLE_COVERAGE,
+  coverageDirectory: "coverage/",
```

Comment:
@ichiriac this is how coverage still works even without `npm run cover`. Try running `COVERAGE=1 npm test`. In CI it works because the `CI` env variable is set.

Reply:
great, didn't knew about that 👍